fingerprints_dns.csv

```diff
@@ -149,6 +149,46 @@ ooni.id_dns_106,isp,,dns,full,202.3.219.209,5,ID,,,
+ooni.in_dns_2,nat,,dns,full,49.205.171.201,10,IN,sinarproject,https://explorer.ooni.org/measurement/20220418T073116Z_webconnectivity_IN_18209_n1_oxSDb3HkQkXMwSMi?input=https%3A%2F%2Fwww.xnxx.com%2F,blockpage_govt
+ooni.in_dns_3,nat,,dns,full,202.83.21.14,10,IN,sinarproject,https://explorer.ooni.org/measurement/20220521T113050Z_webconnectivity_IN_131269_n1_Pv0UoubIMBLmdl8L?input=https%3A%2F%2Fwww.wechat.com%2F,blockpage_govt
+ooni.in_dns_4,nat,,dns,full,49.207.46.62,10,IN,sinarproject,https://explorer.ooni.org/measurement/20220627T010435Z_webconnectivity_IN_18209_n1_tLROFT2RmqWhLxdM?input=http%3A%2F%2Fbdcburma.wordpress.com%2F2012%2F07%2F27%2Fphotos-about-burmese-buddhists-killed-the-muslims-in-burma-2%2F,blockpage shown in OONI measurement
+ooni.in_dns_5,nat,,dns,full,103.206.163.150,10,IN,sinarproject,https://explorer.ooni.org/measurement/20220530T185416Z_webconnectivity_IN_135031_n1_dGgyfkCKF6Po0KQ3?input=http%3A%2F%2Fmicous.com%2F,blockpage shown in OONI measurement
+ooni.in_dns_6,isp,,dns,full,49.44.79.236,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220526T221323Z_webconnectivity_IN_55836_n1_iKD1mhrh8oqG9J7U?input=https%3A%2F%2Fwww.bigo.tv%2F,"timed out, local ISP"
+ooni.in_dns_7,isp,,dns,full,59.185.3.14,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220627T211811Z_webconnectivity_IN_17813_n1_sGAcsBScFgAhfeB5?input=http%3A%2F%2Finfluence.in%2F,"timed out, local ISP"
+ooni.in_dns_8,isp,,dns,full,202.83.21.15,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220114T023036Z_webconnectivity_IN_24309_n1_AHs2ako5D4Xu9w8N?input=https%3A%2F%2Fwww.weiyun.com%2F,"timed out, local ISP"
+ooni.in_dns_9,isp,,dns,full,202.83.24.75,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220216T151340Z_webconnectivity_IN_24309_n1_I3wQRGFxx6yRqvWA?input=https%3A%2F%2Ffunscrape.com%2F,"timed out, local ISP"
+ooni.in_dns_10,isp,,dns,full,218.248.112.60,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220606T113115Z_webconnectivity_IN_9829_n1_pkWnR4H5UrDGkcFo?input=http%3A%2F%2Fwww.parallel-app.com%2F,"timed out, local ISP"
+ooni.in_dns_11,vbw,,dns,full,10.40.124.12,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220417T011800Z_webconnectivity_IN_55824_n1_p2b2sLBjFMEjdfIn?input=https%3A%2F%2Findiansexstories.net%2F,"timed out, private IP"
+ooni.in_dns_12,vbw,,dns,full,10.40.124.13,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220418T041741Z_webconnectivity_IN_55824_n1_73HY9sL4HmAndyB9?input=https%3A%2F%2Feztv.re%2F,"timed out, private IP"
+ooni.in_dns_13,vbw,,dns,full,10.197.186.13,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220418T041741Z_webconnectivity_IN_55824_n1_73HY9sL4HmAndyB9?input=https%3A%2F%2Feztv.re%2F,"timed out, private IP"
+ooni.in_dns_14,vbw,,dns,full,10.197.187.13,5,IN,sinarproject,https://explorer.ooni.org/measurement/20220418T041741Z_webconnectivity_IN_55824_n1_73HY9sL4HmAndyB9?input=https%3A%2F%2Feztv.re%2F,"timed out, private IP"

```

Comment:
These similarly are also only present in India data, in these ISPs: 4758,  55824, 141340. I'm making the into isp level  blocks.